packages/turf-isolines/index.d.ts

```diff
@@ -1,11 +1,8 @@
-/// <reference types="geojson" />
-
-type Points = GeoJSON.FeatureCollection<GeoJSON.Point>;
-type LineStrings = GeoJSON.FeatureCollection<GeoJSON.LineString>;
+import {Points, MultiLineStrings} from '@turf/helpers'
 
 /**
  * http://turfjs.org/docs/#isolines
  */
-declare function isolines(points: Points, z: string, resolution: number, breaks: Array<number>): LineStrings;
+declare function isolines(points: Points, breaks: Array<number>, property?: string): MultiLineStrings;
```

Comment:
Looks like you're missing `options`

You would need to build an interface for that.

```typescript
interface Options {
  isolineProperties: any[],
  commonProperties: any
}
```
Then declare the option as a param.